ewatercycle/parametersetdb/config.py

```diff
@@ -27,7 +28,8 @@ class AbstractConfig(ABC):
         config (dict): Content of config as Python dictionary
 
     """
-    @abstractmethod
+    config: Any = None
+
```

Comment:
Adding config as class variable make Sphinx doc generator give following warning
```
.../ewatercycle/parametersetdb/config.py:docstring of ewatercycle.parametersetdb.config.AbstractConfig.config:1: WARNING: duplicate object description of ewatercycle.parametersetdb.config.AbstractConfig.config, other instance in apidocs/ewatercycle.parametersetdb.config, use :noindex: for one of them
```

The docs warning can be fixed by removing the class docstring and by giving config its own docstring
```suggestion
    config: Any = None
    """Content of config as Python dictionary"""

```